<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageButtion</class>
 <widget class="QWidget" name="imageButtion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="imagePressButton" name="widget1" native="true"/>
   </item>
   <item>
    <widget class="imagePressButton" name="widget2" native="true"/>
   </item>
   <item>
    <widget class="imagePressButton" name="widget3" native="true"/>
   </item>
   <item>
    <widget class="imagePressButton" name="widget4" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imagePressButton</class>
   <extends>QWidget</extends>
   <header>imagepressbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
